schema.sql

```diff
@@ -41,4 +41,15 @@ CREATE TABLE visits (
     animal_id INTEGER REFERENCES animals(id),
     vet_id INTEGER REFERENCES vets(id),
     date_of_visit DATE
-);
\ No newline at end of file
+);
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+
+CREATE INDEX animal_index ON visits(animal_id);
+CREATE INDEX vet_index ON visits(vet_id);
+CREATE INDEX email_index ON owners(email);
```

Comment:
- [ ] [OPTIONAL] This is a nice-to-have a suggestion as it won't improve the query that was reported as problematic in the requirements for this project 😃 

As email addresses are not case sensitive, we might need to look for the emails in a case insensitive way. There is an option to reflect that in the index by using:

```
CREATE INDEX ON owners ((lower(email)));
```

Reply:
wow i wasn't aware of this thanks i will add that to the project. Thanks @mukhammadrizooff